pulsar-common/src/main/java/org/apache/pulsar/common/util/netty/EventLoopUtil.java

```diff
@@ -35,43 +35,61 @@
 import io.netty.channel.socket.nio.NioDatagramChannel;
 import io.netty.channel.socket.nio.NioServerSocketChannel;
 import io.netty.channel.socket.nio.NioSocketChannel;
+import io.netty.incubator.channel.uring.IOUring;
+import io.netty.incubator.channel.uring.IOUringDatagramChannel;
+import io.netty.incubator.channel.uring.IOUringEventLoopGroup;
+import io.netty.incubator.channel.uring.IOUringServerSocketChannel;
+import io.netty.incubator.channel.uring.IOUringSocketChannel;
 import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.ThreadFactory;
 import lombok.extern.slf4j.Slf4j;
 import org.apache.bookkeeper.common.util.affinity.CpuAffinity;
+import org.apache.commons.lang3.StringUtils;
 
 @SuppressWarnings("checkstyle:JavadocType")
 @Slf4j
 public class EventLoopUtil {
 
+    private static final String ENABLE_IO_URING = "enable.io_uring";
```

Comment:
I'm wondering whether we need a prefix here, or if it's the same as how netty does it.

That said, if it's a Pulsar specific property, name it as:

```java
private static final String ENABLE_IO_URING_KEY = "pulsar.enable.io_uring";
```

can avoid accidental conflicts.

Reply:
Sorry for missing this comment. This is a good idea, so like: `pulsar.enable.io_uring`.

Reply:
I learn more properties don't separate `enable` and the target with a `.`. But it can be a style preference :)

Reply:
Good suggestion, I will push a new PR to improve it.

Reply:
@nodece Please also help to add labels and milestone.